src/test/java/oss/fosslight/controller/ProjectIdentificationReviewConfirmIntegrationTest.java

```diff
@@ -0,0 +1,118 @@
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc(addFilters = false)
+@SpringBootTest
+@Transactional
+public class ProjectIdentificationReviewConfirmIntegrationTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    private String prjId;
+
+    @BeforeEach
+    void createProject() throws Exception {
+        MockHttpServletResponse response = mockMvc.perform(post("/project/saveAjax")
+                .param("copy", "false")
+                .param("prjName", UUID.randomUUID().toString())
+                .param("prjVersion", "v1")
+                .param("noticeType", "10")
+                .param("distributionType", "10")
+                .param("comment", "<p>test</p>")
+                .param("osType", "100")
+                .param("priority", "30")
+                .param("statusRequestYn", "")
+                .param("listId", "")
+                .param("publicYn", "Y")
+                .param("networkServerType", "N")
+                .param("prjDivision", "999")
+                .param("prjUserId", "test2")
+                .param("prjId", "")
+                .param("prjDivision", "")
+                .param("distributeTarget", "NA")
+                .param("prjModelJson", "[]")
+                .header("user-agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andReturn().getResponse();
+
+        Map<String, Map<String, String>> responseMapForProject = new ObjectMapper().readValue(response.getContentAsString(), Map.class);
+        prjId = responseMapForProject.get("resultData").get("prjId");
+
+        String content = "{\"prjId\":\"" + prjId + "\",\"identificationStatus\":\"REV\"}";
+        mockMvc.perform(post("/project/updateProjectStatus")
+                .content(content)
+                .header("user-agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @DisplayName("Identification Review Confirm should be success When parameter is valid")
+    void identificationReviewConfirmShouldBeSuccess() throws Exception {
+        //given
+        String content = "{\"prjId\":\"" + prjId + "\",\"identificationStatus\":\"CONF\"}";
+
+        //when
+        mockMvc.perform(post("/project/updateProjectStatus")
+                .content(content)
+                .header("user-agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36")
```

Comment:
User agnet header can be declared as static final at the start of the class
```java
private static fianl String USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.107 Safari/537.36"
```

Please remember upper case. It is convention for Java to declare a constant .

Reply:
Thank you! I fixed the code and merged commits into one.